src/main/java/io/pravega/perf/ReaderWorker.java

```diff
@@ -54,20 +56,25 @@ private Performance createBenchmark() {
 
     @Override
     public Void call() throws InterruptedException, ExecutionException, IOException {
-        perf.benchmark();
+        try {
+            perf.benchmark();
+        } catch (Exception e) {
+            e.printStackTrace();
```

Comment:
What's the point of printing the stack trace only to rethrow the same one?

Reply:
actually the callables (even though i have them printing in PravegaperTest class) are not printing the stack, so, this is alternative